pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -14,16 +14,21 @@ import (
 )
 
 const (
-	trivyIgnore = ".trivyignore"
+	DefaultIgnoreFile = ".trivyignore"
 )
 
 var (
 	sources = []string{Nvd, RedHat, Debian, DebianOVAL, Alpine,
 		RubySec, RustSec, PhpSecurityAdvisories, NodejsSecurityWg, PythonSafetyDB}
 )
 
-func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreUnfixed bool) []DetectedVulnerability {
-	ignoredIDs := getIgnoredIDs()
+func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreUnfixed bool, ignoreFile string) []DetectedVulnerability {
+	var ignoredIDs []string
+	if ignoreFile == "" {
+		ignoredIDs = getIgnoredIDs(DefaultIgnoreFile)
+	} else {
+		ignoredIDs = getIgnoredIDs(ignoreFile)
+	}
```

Comment:
```suggestion
	ignoredIDs := getIgnoredIDs(ignoreFile)
```
I think we don't have to check if ignoreFile is empty since the default value is assigned by `cli.StringFlag`. In short, this value will not be empty unless a user specifies the empty value.

Reply:
@knqyf263 
Thank you for a very detailed review.
I understood what you said.
I modify as suggested.

Reply:
@knqyf263 
CircleCI was failed
I open this conversation again

Reply:
@knqyf263 
I remove ignoreFile check completely
and CircleCI was success
so, this conversation is closed